### DISCUSSION TITLE
Building antlr from source

### DISCUSSION THREAD

**Discussion by chris-miner** [antlr/antlr4#3897] [General]
Is this the place to ask questions about building antlr from source.  For example if you wanted a development build for the purpose of submitting pull requests?

Or if I ran into issues with steps outlined in these docs?

https://github.com/antlr/antlr4/blob/master/doc/building-antlr.md
https://github.com/antlr/antlr4/blob/master/doc/antlr-project-testing.md

**Comment by KvanTTT** [comment]
Yes, feel free ask any questions in discussions. I also recommend StackOverflow as well.

**Reply by chris-miner** [reply]
Thanks for that tip about SO.  I checked there yesterday, but my query was wrong so I only saw posts from like 2015.  I see now what I did wrong and found current questions.
